### PR TITLE
Install swsssdk-py3 in the base Debian image for python3 based apps

### PR DESCRIPTION
**Pull request by rajendra-dendukuri**
Install swsssdk-py3 in the base Debian image for python3 based apps

**- Why I did it**
To use the swsssdk in python3 based apps.

**- How I did it**
Install swsssdk-py3 in the base Debian image.

**- How to verify it**
Use swsssdk-py3 in a sample python3 app to access ConfigDB.

**- Description for the changelog**

Install swsssdk-py3 in the base Debian image

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: rules/sonic-ztp.mk [hunk 1]
```diff
@@ -5,6 +5,7 @@ SONIC_ZTP_VERSION = 1.0.0
 
 SONIC_ZTP = sonic-ztp_$(SONIC_ZTP_VERSION)_all.deb
 $(SONIC_ZTP)_SRC_PATH = $(SRC_PATH)/sonic-ztp
+$(SONIC_ZTP)_WHEEL_DEPENDS +=  $(SWSSSDK_PY3)
```
Comment (jleveque) [inline rules/sonic-ztp.mk:8]: Remove extra space after `+=`

**Comment by rajendra-dendukuri** [comment]
retest broadcom please

**Comment by rajendra-dendukuri** [comment]
retest vsimage please

**Comment by lguohan** [comment]
retest vsimage please

**Comment by rajendra-dendukuri** [comment]
retest vsimage please

**Comment by rajendra-dendukuri** [comment]
@jleveque can you please merge this change. I have more PR's following which will need this change.

File: files/build_templates/sonic_debian_extension.j2 [hunk 2]
```diff
@@ -116,6 +116,14 @@ sudo cp {{swsssdk_py2_wheel_path}} $FILESYSTEM_ROOT/$SWSSSDK_PY2_WHEEL_NAME
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install $SWSSSDK_PY2_WHEEL_NAME
 sudo rm -rf $FILESYSTEM_ROOT/$SWSSSDK_PY2_WHEEL_NAME
 
+# Install SwSS SDK Python 3 package
+if [ -e {{swsssdk_py3_wheel_path}} ]; then
```
Comment (lguohan) [inline files/build_templates/sonic_debian_extension.j2:120]: why is this needed?

Comment (rajendra-dendukuri) [reply files/build_templates/sonic_debian_extension.j2:120]: If the packages requesting swsssdk_py3 whl are not included in the build, the file may not exist. It is a safety check.

Comment (lguohan) [reply files/build_templates/sonic_debian_extension.j2:120]: if the path is not there, then we should fail the build because we need to install the pacakge.

Comment (rajendra-dendukuri) [reply files/build_templates/sonic_debian_extension.j2:120]: Let us assume ZTP package is the only one that requires py3. If ENABLE_ZTP is not set, the py3 package is not created. But that doesn't mean the build should fail.

File: rules/sonic-ztp.mk (hunk 1 above)
Comment (rajendra-dendukuri) [reply rules/sonic-ztp.mk]: @jleveque Request change has been done can you please mark it as complete.
